dist/assets/css/custom.css

```diff
@@ -289,19 +289,29 @@
 .portfolio-1 .portfolio-item .portfolio-link .details .description {
     background-color: #3e444d;
     color: white;
-    display: table;
-    height: 30px;
+    /*display: table;*/
```

Comment:
Can we delete this instead of commenting it out?  It is easy to add back later, and I would rather not have dead code lying around.